### PR TITLE
[pre-commit] Configure ruff to sort import in the doc examples

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |

## Description

Faster, but I'm not sure if ruff handle first party / third parties correctly. It's good enough for the doc example anyway, and the per file configuration is better than a regex...

### CODE REVIEW

File: requirements_test_pre_commit.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ pre-commit~=3.1;python_version>='3.8'
 bandit==1.7.4
 black==23.1.0
 flake8==6.0.0;python_version>='3.8'
-ruff
+ruff==0.0.254
```
Comment (DanielNoord) [inline requirements_test_pre_commit.txt:7]: I'd rather just have this install latest right? Pinning feels unnecessary as `pre-commit` will get the version as specified anyway.

Comment (Pierre-Sassoulas) [reply requirements_test_pre_commit.txt:7]: Yeah, I'm fed up with approving dependabot's dozen of MR too.

Comment (Pierre-Sassoulas) [reply requirements_test_pre_commit.txt:7]: Should I do another commit for that, or change this in another MR?

Comment (DanielNoord) [reply requirements_test_pre_commit.txt:7]: Can we just remove this file perhaps?

Comment (DanielNoord) [reply requirements_test_pre_commit.txt:7]: Let's do a separate commit in this PR

Comment (Pierre-Sassoulas) [reply requirements_test_pre_commit.txt:7]: We had this argument with Marc, he wanted to have the possibility to use tools without pre-commit, I did not like the duplication personally. I'm not sure about pinging him for that though.

Comment (DanielNoord) [reply requirements_test_pre_commit.txt:7]: With dependabot we are on the latest version anyway in `pre-commit`. Specifying the versions in a file does not improve the workflow significantly as you would still need to `pip install` yourself manually.

Specifying the tools we use seems fine, pinning them to versions and having to update seems a timewaste

Comment (Pierre-Sassoulas) [reply requirements_test_pre_commit.txt:7]: So let's keep the file and not pin then. Do you think we need another MR of should I just fix this here ?

Comment (DanielNoord) [reply requirements_test_pre_commit.txt:7]: Let's fix here

**Review by DanielNoord — APPROVED** [review]

**Pull request merged**